crates/genesis/src/lib.rs

```diff
@@ -585,6 +585,25 @@ impl ChainConfig {
     fn is_active_at_timestamp(&self, config_timestamp: Option<u64>, timestamp: u64) -> bool {
         config_timestamp.is_some_and(|cb| cb <= timestamp)
     }
+
+    /// Returns the blob schedule for the chain.
+    pub fn blob_schedule(&self) -> HardforkBlobParams {
+        let mut schedule = HardforkBlobParams {
+            cancun: self.blob_schedule.get("cancun").cloned().unwrap_or_default(),
+            prague: self.blob_schedule.get("prague").cloned().unwrap_or_default(),
```

Comment:
if those are missing we need to use `BlobParams::cancun` and `BlobParams::prague`